.github/workflows/detect-breaking-changes-report.yml

```diff
@@ -122,40 +113,36 @@ jobs:
     # This is very weird, the actual request goes through (comes back with a 201), but does not assign the team.
```

Comment:
The reasons for that might be either:
- GITHUB_TOKEN has not enough permissions (https://github.community/t/bug-github-action-cannot-assign-teams/131186/3?u=tolzhabayev)
- Additionally it might be that it should be called just "plugins-platform-frontend" https://github.community/t/bug-github-action-cannot-assign-teams/131186/9?u=tolzhabayev

Reply:
Thanks! Yeah, unfortunately I tried all type of team handles (even `plugins-platform-frontend`), but no luck.
The GITHUB_TOKEN could be an issue indeed, although I was testing it with the `pull_request` event which had permissions to assign labels or comment on the PR. 🤷‍♂️